sold.cc

```diff
@@ -496,9 +496,90 @@ void Sold::RelocateSymbol_x86_64(ELFBinary* bin, const Elf_Rel* rel, uintptr_t o
             break;
         }
 
-        // TODO(akawashiro) Fill here with some exaplanation.
-        case R_X86_64_DTPMOD64:
-        case R_X86_64_DTPOFF64:
+        // TODO(akawashiro) Handle TLS variables in executables.
+        case R_X86_64_DTPMOD64: {
+            const std::string name = bin->Str(sym->st_name);
+            uintptr_t index = syms_.ResolveCopy(name, soname, version_name);
+            newrel.r_info = ELF_R_INFO(index, type);
+
+            if (bin->tls() == NULL) {
```

Comment:
Probably we will want to factor out this code when we support another architecture like aarch64, but this is fine for now.

Reply:
I get it. Thank you.